Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -278,6 +278,17 @@ public ConnectionMode ConnectionMode
         /// If this is not set the database account consistency level will be used for all requests.
         /// </summary>
         public ConsistencyLevel? ConsistencyLevel { get; set; }
+
+        /// <summary>
+        /// Sets the PriorityLevel for requests created using cosmos client.
+        /// If PriorityLevel is also specified at request level in RequestOptions, that PriorityLevel takes precedence.

```

Comment:
Prefer linking to the class instead of just using the name

```suggestion
        /// If PriorityLevel is also specified at request level in <see cref="RequestOptions.PriorityLevel"/>, that PriorityLevel takes precedence.

```

Reply:
Used cref